static/assets/js/happyfoxchat.js

```diff
@@ -1,18 +0,0 @@
-<!-- HappyFox Live Chat Script -->
```

Comment:
Yea I have to put these things back in. Essentially the assets folder gets deleted and rebuilt on the build, so I have to copy these files over after. Shouldn't be too hard.